### DISCUSSION TITLE
HTML-CSS-JAVASCRIPT-WEB-COMPONENT

### DISCUSSION THREAD

**Discussion by Shubham56-droid** [Shubham56-droid/CSS-ANIMATION-PROJECTS#2] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/Shubham56-droid/HTML-CSS-JAVASCRIPT-ANIMATION/releases/tag/'>HTML-CSS-JAVASCRIPT-WEB-COMPONENT</a>.</em>

**Comment by MeMohamedSulaiman** [comment]
So What do ya want to discuss
